pkg/pipelines/statustracker/deployment.go

```diff
@@ -108,6 +112,10 @@ func makeEnvironment(repoURL, driver string) []corev1.EnvVar {
 			Name:  "OPERATOR_NAME",
 			Value: operatorName,
 		},
+		{
+			Name:  "STATUS_TRACKER_SECRET",
+			Value: "git-host-access-token",
```

Comment:
Use this const CommitStatusTrackerSecret?

Reply:
Anywhere using this "git-host-access-token" string should use the constant?  If the secret constant shall be used not limited to commit status tracker, consider renaming the const.